syntax = "proto3";

package io.github.slackhappy.sds.examples;

message WikipediaViews {
    int32 count = 1;
}